### PR TITLE
Fix handler rules for OSS-Fuzz.

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

File: src/appengine/handlers/bots.py [hunk 1]
```diff
@@ -90,6 +90,7 @@ def _get_host_workers_heartbeats():
 class Handler(base_handler.Handler):
   """Handler that gets the bot list."""
 
+  @handler.check_admin_access_if_oss_fuzz
   @handler.check_user_access(need_privileged_access=False)
```
Comment (Dor1s) [inline src/appengine/handlers/bots.py]: Is this line still needed in all of these?

Comment (inferno-chromium) [reply src/appengine/handlers/bots.py]: Yes above condition only exclude oss-fuzz.

**Pull request merged**